man/org.freedesktop.hostname1.xml

```diff
@@ -306,9 +310,11 @@ node /org/freedesktop/hostname1 {
     to the <varname>IMAGE_ID=</varname> and <varname>IMAGE_VERSION=</varname> fields of the
     <filename>os-release</filename> file.</para>
 
-    <para><varname>HardwareVendor</varname> and <varname>HardwareModel</varname> expose information about the
-    vendor of the hardware of the system. If no such information can be determined these properties are set
-    to empty strings.</para>
+    <para><varname>HardwareVendor</varname>, <varname>HardwareModel</varname> and <varname>HardwareSKU</varname>
```

Comment:
Not a strict rule, but we usually insert comma before `and`.

Reply:
I was unsure, since both are used. I changed it.